en/views/helpers/rss.rst

```diff
@@ -184,7 +184,7 @@ body of your blog. In the code above we used ``strip_tags()`` and
 as they could cause validation errors. Once we have set up the data for the
 feed, we can then use the :php:meth:`RssHelper::item()` method to create the XML
 in RSS format. Once you have all this setup, you can test your RSS feed by going
-to your site ``/posts/index.rss`` and you will see your new feed. It is always
+to your site ``/articles/index.rss`` and you will see your new feed. It is always
```

Comment:
Isnt this usually /articles.rss without action?
I am not sure though.

Reply:
Can confirm it will work as
articles/index.rss
or
articles.rss

Should I create a new pull with something to that effect in it as well?

Reply:
You can also fix in this one